src/Domain/Entities/Items/UserItem.cs

```diff
@@ -10,15 +10,15 @@ public class UserItem : AuditableEntity
 {
     public int Id { get; set; }
     public int UserId { get; set; }
-    public string BaseItemId { get; set; } = string.Empty;
+    public string ItemId { get; set; } = string.Empty;
```

Comment:
Missed that one but don't forget this requires a migration

Reply:
was planning to do one migration at the end of the 8 Pullrequest, as we won't patch before heirloom are completely done. Do you want a migration each pull request? Even if it's going to stack up migrations that will have to be done one after the other?

Reply:
and i don't understand , why are you saying i missed it?

Reply:
oh i think you mean you missed it.

Reply:
Yes otherwise this commit would break the application